src/methods.jl

```diff
@@ -10,11 +10,11 @@ derivatives.
 const DEFAULT_CONDITION = 100
 
 """
-    FiniteDifferences.TINY
+    add_tiny(x::Real)
 
-A tiny number added to some quantities to ensure that division by 0 does not occur.
+Add a tiny number, 10^{-20}, to `x` while preserving its type.
 """
-const TINY = 1e-20
+add_tiny(x::T) where {T<:Real} = x + T(1e-20)
```

Comment:
```suggestion
add_tiny(x::T) where {T<:Real} = x + convert(T, 1e-20)
```

more formally i think `convert` belongs here rather than a constructor.